package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "yaml-to-analytics",
-  "version": "0.0.0",
+  "version": "0.1.0",
```

Comment:
What happened to 0.0.1? :sweat_smile:

Reply:
This changes the user interface so should bump the bigger version, I think. The current release is 0.0.10.